app/assets/stylesheets/components/_ppm-estimator.scss

```diff
@@ -59,16 +59,98 @@
   padding-top: 1.7rem;
 }
 
+.ppm-estimate-header {
+  font-size: $h2-font-size;
+}
+
+.ppm-estimate-print {
+  float: right;
+}
+
 .ppm-estimate-results {
   border-top: 0.125rem solid #4a4a4a;
   margin-top: 1.7rem;
   padding-top: 2rem;
+  -webkit-print-color-adjust: exact;
```

Comment:
Here's the magic WebKit- and Blink- specific CSS that tells the browser to include background images and colors by default when printing. Unfortunately, users of other browsers will need to manually check that box if they want the output to look right ATM.

Reply:
Interesting! I didn't know about this property (non-standard though it may be). Linking to [the MDN documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-print-color-adjust) for future reference.